### DISCUSSION TITLE
Model Error during perfrom message_ix -> tutorial ->westeros_baseline

### DISCUSSION THREAD

**Discussion by Samtsy1** [iiasa/message_ix#512] [MESSAGE modeling]
I ran into this problem while dealing with the "Time to Solve the Model" phase
In [34]: scenario.solve()

`**ModelError                                Traceback (most recent call last)**
~\AppData\Local\Temp/ipykernel_10264/2994721627.py in <module>
----> 1 scenario.solve()

~\anaconda3\envs\message_env\lib\site-packages\message_ix\core.py in solve(self, model, solve_options, **kwargs)
    563             :class:`.GAMSModel`.
    564         """
--> 565         super().solve(model=model, solve_options=solve_options, **kwargs)
    566 
    567     def add_macro(self, data, scenario=None, check_convergence=True, **kwargs):

~\anaconda3\envs\message_env\lib\site-packages\ixmp\core.py in solve(self, model, callback, cb_kwargs, **model_options)
   1666         # **Iterate until convergence**
   1667         while True:
-> 1668             model.run(self)
   1669 
   1670             # **Store an iteration number to help the callback**

~\anaconda3\envs\message_env\lib\site-packages\message_ix\models.py in run(self, scenario)
    361 
    362         try:
--> 363             result = super().run(scenario)
    364         finally:
    365             # **Remove the optfile regardless of whether the run completed**

~\anaconda3\envs\message_env\lib\site-packages\ixmp\model\gams.py in run(self, scenario)
    294         except CalledProcessError as exc:
    295             # **Do not remove self.temp_dir; the user may want to inspect the GDX file**
--> 296             raise self.format_exception(exc, model_file) from None
    297 
    298         # **Read model solution**

**ModelError:** GAMS errored with return code 7:
    There was a licensing error

For details, see the terminal output above, plus:
Listing   : C:\Users\tsy_s\anaconda3\envs\message_env\lib\site-packages\message_ix\model\MESSAGE_run.lst
Input data: C:\Users\tsy_s\anaconda3\envs\message_env\lib\site-packages\message_ix\model\data\MsgData_Westeros_Electrified_baseline.gdx

`
![1](https://user-images.githubusercontent.com/91485839/134943748-e1c63062-ad0e-473c-bcbf-963d914dadb5.png)
![3](https://user-images.githubusercontent.com/91485839/134943754-040215da-1fa7-41d0-9873-d07de19fee14.png)

Hope to get your help, 
Thank you.

**Comment by behnam-zakeri** [comment]
Thanks @Samtsy1 for reaching out. As your error messages shows, there is a problem with your GAMS license. Please be aware that new versions of GAMS don't support a free license even for small problems. For more information, you may refer to [MESSAGEix installation notes](https://docs.messageix.org/en/stable/install.html#gams-required) on this, consult with your colleagues on what type of license you are using in your institution, or you can request a free [demo license from GAMS](https://www.gams.com/36/docs/UG_License.html).

**Reply by Samtsy1** [reply]
Thank you for reminding me. It was really my fault. I installed the latest version of GAMS.

I'll try again. Thank you for your help.
